insights/parsers/pmrep.py

```diff
@@ -58,4 +58,4 @@ def search(self, **kwargs):
             >>> pmrep_doc_obj.search(name__endswith='swap.pagesout')
             [{'name': 'swap.pagesout', 'value': '5.000'}]
         """
-        return keyword_search(self, **kwargs)
+        return sorted(keyword_search(self, **kwargs), key=lambda x: x['name'])
```

Comment:
@wushiqinlou - It's not a good idea to sort the result inner the `search`. Although it is caused by the different sort-policies in py2 and py3, it should be fixed in the test too.

Please re-sort the result in the test of the example of the search again.